compiler/src/iree/compiler/Codegen/Common/test/decompose_softmax.mlir

```diff
@@ -1,43 +1,84 @@
-// RUN: iree-opt --split-input-file --iree-codegen-decompose-softmax -cse %s | FileCheck %s
+// RUN: iree-opt --split-input-file --iree-codegen-decompose-softmax -cse %s | FileCheck %s --check-prefix=CHECK-FUSE
+// RUN: iree-opt --split-input-file --iree-codegen-decompose-softmax=use-fusion=false -cse %s | FileCheck %s --check-prefix=CHECK-FUSE
+// RUN: iree-opt --split-input-file --iree-codegen-decompose-softmax=use-fusion=true -cse %s | FileCheck %s --check-prefix=CHECK-NO-FUSE
```

Comment:
l.1 and l.3 are the same, can we keep the first one? Also, lets not use `check-prefix` for default?